src/_includes/site_nav.hbs

```diff
@@ -0,0 +1,20 @@
+<nav class="site-nav headroom">
+  <div class="content">
+    <div class="sub-nav-left">
+      <a href="/"><img class="logo" src="/img/logos/localhost.png"/></a>
+      <div class="sub-nav-control visible-md down"></div>
+    </div>
+    <div class="hidden-md hello">
+      <ul class="sub-nav-right">
+        {{> site_nav_links }}
+      </ul>
+    </div>
+
+    <div class="visible-md">
+      <div class="current-active"></div>
+      <ul class="sub-nav-right mobile">
```

Comment:
same as below. at first glance it's not clear that this is the sub-nav-right on mobile because of the different element structure